packages/lodestar/src/network/peers/discover.ts

```diff
@@ -226,13 +226,18 @@ export class PeerDiscovery {
       this.metrics?.discovery.findNodeQueryRequests.inc({action: "start"});
     }
 
-    const randomNodeId = crypto.randomBytes(64).toString("hex");
-
     this.randomNodeQuery = {code: QueryStatusCode.Active, count: 0};
     const timer = this.metrics?.discovery.findNodeQueryTime.startTimer();
 
     try {
-      const enrs = await this.discv5.findNode(randomNodeId);
+      // async version of crypto.randomBytes
```

Comment:
Consider using https://nodejs.org/api/util.html#utilpromisifyoriginal (this dependency doesn't work in browser but this module isn't expected to run in browser)